lms/djangoapps/courseware/module_render.py

```diff
@@ -642,6 +643,8 @@ def rebind_noauth_module_to_user(module, real_user):
 
     system.set(u'user_is_staff', user_is_staff)
     system.set(u'user_is_admin', has_access(user, u'staff', 'global'))
+    system.set(u'user_is_beta', CourseBetaTesterRole(course_id).has_user(user))
```

Comment:
@symbolist ```has_access(user, 'beta_tester', course_descriptor, course_key)``` this functionality is not available for ```beta_tester```, I used same as in other places.